### DISCUSSION TITLE
Can you help verify our bridged token contract on Arbiscan?

### DISCUSSION THREAD

**Discussion by magpiexyz** [wormhole-foundation/wormhole#2535] [General] [answered]
We have recently bridged our MGP token from BSC to Arbitrum. Can you help verify our token contract on Arbiscan?
https://arbiscan.io/token/0xa61f74247455a40b01b0559ff6274441fafa22a3#code

Thanks in advance!

**Comment by bruce-riley** [comment]
This document describes how to verify the token:
https://github.com/wormhole-foundation/wormhole/blob/main/ethereum/docs/token_verification.md

I tried running through the process but was unable to verify that the token is a proxy. I opened a ticket with Arbiscan (ID#349161) and will post here when I get a response.

**Comment by bruce-riley** [comment, marked as answer]
@kcsongor was able to verify the token on arbiscan. We will have to wait for the issue mentioned above to be resolved by Arbitrum before we can mark it as a proxy contract.

**Comment by aadam-10** [comment]
closing discussion as the issue seems to be resolved
